From 2a7b8d84520fc98c4be46cbf188d5fb26ade7fe0 Mon Sep 17 00:00:00 2001
From: Ihar Hrachyshka <ihrachys@redhat.com>
Date: Tue, 9 Jan 2018 12:12:48 -0800
Subject: [PATCH 118/155] Switch rally job from q-* to neutron-* service names

When at least one service named as q-* is present in ENABLED_SERVICES,
then devstack utilizes lib/neutron-legacy to configure services
regardless of how other services are deployed (e.g. with lib/neutron).
This breaks deployment using lib/neutron.

Switching to new names doesn't change anything substantial because
devstack plugin equally handles both variants. It allows to use new
devstack neutron library though.

Change-Id: Id0d35523651131766a70e78bf130205c1c63acd5
(cherry picked from commit 661247a61708d9ceb716e0778ec0c83345ddd3ba)
---
 playbooks/legacy/neutron-rally-neutron/run.yaml | 2 +-
 rally-jobs/extra/trunk_scenario.setup           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/playbooks/legacy/neutron-rally-neutron/run.yaml b/playbooks/legacy/neutron-rally-neutron/run.yaml
index 2335b48..a9f7e7b 100644
--- a/playbooks/legacy/neutron-rally-neutron/run.yaml
+++ b/playbooks/legacy/neutron-rally-neutron/run.yaml
@@ -66,7 +66,7 @@
           ENABLED_SERVICES+=cinder,c-api,c-vol,c-sch,c-bak,
           ENABLED_SERVICES+=g-api,g-reg,
           ENABLED_SERVICES+=n-api,n-crt,n-cpu,n-sch,n-cond,
-          ENABLED_SERVICES+=q-qos,
+          ENABLED_SERVICES+=neutron-qos,
 
           if [ $DEVSTACK_GATE_SWIFT -eq 1 ];
           then
diff --git a/rally-jobs/extra/trunk_scenario.setup b/rally-jobs/extra/trunk_scenario.setup
index a2af360..4b3e9bb 100644
--- a/rally-jobs/extra/trunk_scenario.setup
+++ b/rally-jobs/extra/trunk_scenario.setup
@@ -1 +1 @@
-enable_service q-trunk
+enable_service neutron-trunk
-- 
2.7.4

